apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/ReleaseController.java

```diff
@@ -29,49 +29,131 @@
 @RequestMapping("/openapi/v1/envs/{env}")
 public class ReleaseController {
 
-  @Autowired
```

Comment:
Is this class file reformatted? There should not be so many changes...

Reply:
And it's better to submit the change in one commit, so that it could be easily tracked in the future.

Thanks!